examples/montage/packages/todomvc-common@13305a5/readme.md

```diff
@@ -0,0 +1,15 @@
+# todomvc-common
+
+> Common TodoMVC utilities used by our apps
+
+
+## Install
+
+```
+$ npm install --save todomvc-common
+```
+
+
+## License
+
+MIT © [TasteJS](http://tastejs.com)
```

Comment:
ok, so I shall remove every file that ends up not being used when the app runs, correct?